modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -1,26 +1,76 @@
+
+You trigger upgrades to the Runtime Fabric components in your cluster from the Anypoint control plane, specifically from Runtime Manager. When you trigger an upgrade, an upgrade message is processed inside the target cluster, and `rtfctl` invokes Helm to upgrade the components' software versions. If you encounter a problem during an upgrade, you can use `rtfctl` to invoke an Helm rollback and restore the cluster to a stable state. A rollback creates a report in the control plane.
+
+== How Application Deployments Work in Runtime Fabric on Self-Managed Kubernetes 
+
+When you deploy an application in Runtime Fabric on Self-Managed Kubernetes, the following occurs:
+
+. You use Runtime Manager to trigger the application deployment.
+. The Runtime Fabric agent in the target cluster receives a request to deploy the application.
+. Runtime Fabric assigns the application's namespace or creates a new namespace if necessary.
+. Runtime Fabric generates the appropriate Kubernetes resources, including a deployment, ConfigMap, secrets, services, and ingress.
+. The Kubernetes deployment resource may include an init container, which fetches dependencies from the Runtime Fabric resource cache.
+. If the resource cache doesn’t contain required dependencies, Runtime Fabric fetches them from the control plane and adds them to the resource cache.
+
+//image 
+
+=== Assigning Namespaces in Runtime Fabric
+
+Each application is deployed into a Kubernetes namespace based on the application’s environment. 
+
+. Runtime Fabric looks for a namespace with the label `rtf.mulesoft.com/envId=<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that label, it looks for a namespace with the name `<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that namespace, it creates a new namespace called `<ANYPOINT_ENVIRONMENT_ID>`.
+
+//add image
+
+=== Monitoring Application Deployments
+
+The Runtime Fabric agent monitors Kubernetes Deployments labelled with `rtf.mulesoft.com/id`. When Kubernetes updates the state of the Deployment, the agent sends that update to the control plane. See xref:logs[Logs] and xref:monitoring[Monitoring] for addtional information. 
+
+
+== Runtime Fabric on Self-Managed Kubernetes: A Shared Responsibility
+
+The successful operation of Anypoint Runtime Fabric on Self-Managed Kubernetes is a shared responsibility. It is critical to understand which areas you provide and manage and which areas MuleSoft provides.
 
 The following image illustrates different MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
 
+//update image
+
 image::runtimefabricresponsibility-self-managed.png[Runtime Fabric on Self-Managed Kubernetes Shared Responsiblities]
 
-* MuleSoft responsibility:
+* MuleSoft provides:
```

Comment:
```suggestion
* MuleSoft responsibility
```

Reply:
I'm going to leave it at provides, as this was some suggested hand-waving from the product team 😬